### ISSUE TITLE
Problems accessing Helios data

### ISSUE THREAD

**Issue by tbloch1** [heliopython/heliopy#647]
Hi,

A few months ago I downloaded some Helios 1 data and everything worked fine.

I'm now trying to get more data, specifically Helios 2, but I get an error.

I took some code from your documentation:
(Note: I tried other date ranges and I made sure HelioPy was up to date).

```python
starttime = datetime(1976, 4, 5, 0, 0, 0)
endtime = starttime + timedelta(hours=12)
probe = '2'
corefit = helios.corefit(probe, starttime, endtime)`
```
This code gives me the following error:

> RuntimeError                              Traceback (most recent call last)
> <ipython-input-8-dd1b765e9af5> in <module>()
>       6 endtime = starttime + timedelta(hours=12)
>       7 probe = '2'
> ----> 8 corefit = helios.corefit(probe, starttime, endtime)
> 
> ~/.conda/envs/tb_env/lib/python3.5/site-packages/heliopy/data/helios.py in corefit(probe, starttime, endtime, verbose, try_download)
>     906     return util.process(dirs, fnames, extension, local_base_dir,
>     907                         remote_base_url, download_func, processing_func,
> --> 908                         starttime, endtime, try_download)
>     909 
>     910 
> 
> ~/.conda/envs/tb_env/lib/python3.5/site-packages/heliopy/data/util.py in process(dirs, fnames, extension, local_base_dir, remote_base_url, download_func, processing_func, starttime, endtime, try_download)
>     129             logger.info('File {}/{}{} not available\n'.format(
>     130                         local_dir, fname, extension))
> --> 131     return timefilter(data, starttime, endtime)
>     132 
>     133 
> 
> ~/.conda/envs/tb_env/lib/python3.5/site-packages/heliopy/data/util.py in timefilter(data, starttime, endtime)
>     153     if len(data) == 0:
>     154         raise RuntimeError(
> --> 155             'No data available between {} and {}'.format(starttime, endtime))
>     156     if isinstance(data, list):
>     157         data = pd.concat(data)
> 
> RuntimeError: No data available between 1976-04-05 00:00:00 and 1976-04-05 12:00:00

**Comment by dstansby** [comment]
I think this is because the Helios archive server is currently down (or at least empty...). This could do with a much better error message to make it clear though, so I'll leave this open.

**Comment by tbloch1** [comment]
Ah I see, possibly related to the shutdown?

**Comment by dstansby** [comment]
🤷‍♂️ who knows. The http site is still working, so I've changed the links to point there where possible. Should be fixed in 0.6.4 which I'll release tonight.
